README.md

```diff
@@ -94,12 +84,39 @@ response_types = ResponseType.Transcript | ResponseType.Captions
     
 client = WebSocketStreamingClient(access_token="ACCESS TOKEN")
 
-response_generator = client.start_stream(media_generator=media_generator,
-                                         media_config=media_config,
-                                         response_types=response_types)
+response_generator = client.start_with_media(media_generator=media_generator,
+                                             media_config=media_config,
+                                             response_types=response_types)
+```
+
+### Providing media via an external source
+
+It is possible to use an external media source to provide media chunks to the Speech Recognition Service.
+To do so, you need to specify the relevant input method when booking the session via Verbit's Ordering API.
+
+In such a scenario, you should **not** provide a media generator to the `WebSocketStreamingClient`. 
+Connecting the `WebSocketStreamingClient` to the Speech Recognition Service will initiate the session
+and signal the server to start listening to the external media source.
```

Comment:
```suggestion
and signal the server to start consuming media from the external media source.
```
just to be ambiguous enough to accommodate for both "listening" and "pulling-from"